docs/topics/http/decorators.txt

```diff
@@ -97,3 +97,21 @@ caching based on specific request headers.
     into account when building its cache key.
 
     See :ref:`using vary headers <using-vary-headers>`.
+
+.. module:: django.views.decorators.cache
+
+Caching
+=======
+
+The decorators in :mod:`django.views.decorators.cache` control server and
+client-side caching.
+
+.. function:: never_cache(view_func)
+
+    This decorator adds headers to ensure the response won’t be cached by
+    browsers or other caches.
```

Comment:
I got this from https://github.com/django/django/blob/4870b304f2eabee3ec761f2d94b6251cae418f49/docs/topics/cache.txt#L1213

I can instead use the same text as for `add_never_cache_headers`.